src/plugins/facebook/facebook.js

```diff
@@ -0,0 +1,62 @@
+/**
+* @title WET-BOEW Facebook embedded page
+* @overview Helps with implementing Facebook embedded pages.
+* @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+* @author @pjackson28
+*/
+( function( $, window, wb ) {
+	"use strict";
+
+	/*
+	* Variable and function definitions.
+	* These are global to the plugin - meaning that they will be initialized once per page,
+	* not once per instance of plugin on the page. So, this is a good place to define
+	* variables that are common to all instances of the plugin on a page.
+	*/
+	var componentName = "wb-facebook",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	fbinited = false,
+
+	// Init facebook API if not inited already
+	initfb = function( ) {
+		if ( !fbinited ) {
+			window.FB.init( {
+				version: "v2.4"
```

Comment:
`version` may be in the query string or at init time (depending on if you want to init fbxml on load or not, which we do not as we want to do it dynamically later as well).  The problem is that we can only init a single version per page and a data-web-facebook-version would be ambiguous if you have two on a page....